src/Microsoft.AspNet.Mvc.TagHelpers/LinkTagHelper.cs

```diff
@@ -152,6 +150,9 @@ private enum Mode
         [Activate]
         protected internal IMemoryCache Cache { get; set; }
 
+        [Activate]
+        protected internal IHtmlEncoder HtmlEncoder { get; set; }
```

Comment:
Need to add a test validating that this encoder is the one that's used in `BuildFallbackBlock` and in `BuildGlobbedLinkTags` (same thing for `ScriptTagHelper`)